src/media/js/main.js

```diff
@@ -1,40 +1,33 @@
-console.log('Sample Commonplace App');
+console.log('Firefox Marketplace Operator Dashboard');
 
-define(
-    'main',
-    [
-        'helpers',  // Must come before mostly everything else.
-        'helpers_local',
-        'forms',  // Comment this if your app has no forms.
-        'l10n',
-        'log',
-        'login',  // Comment this if your app does not have accounts.
-        'navigation',
-        'operators',
-        'regions',
-        'settings',
-        'templates',
-        'user',  // Comment this if your app does not have accounts.
-        'z'
-    ],
-function() {
-    var console = require('log')('main');
+define('main', ['init'], function() {
+require([
+    'helpers_local',
```

Comment:
Yeah. ﻿I sometimes get confused on what goes in init.js and what goes in main.js.